src/AVFoundation/AVLayerVideoGravity.cs

```diff
@@ -67,7 +69,9 @@ static internal NSString EnumToKey (AVLayerVideoGravity vg)
 			case AVLayerVideoGravity.Resize:
 				return AVPlayerLayer.GravityResize;
 			default:
-				return null;
+				// The default value according to docs is AVLayerVideoGravity.ResizeAspect so return AVPlayerLayer.GravityResizeAspect;
```

Comment:
The default value _for the VideoGravity property_ is GravityResizeAspect, but this method (EnumToKey) is being called from other places as well, so you can't assume that GravityResizeAspect will be the default value for all callers.

I think I would just change this default case to:

```cs
return null!:
```